redis/src/main/scala/com/avsystem/commons/redis/actor/SentinelsMonitoringActor.scala

```diff
@@ -15,7 +16,8 @@ final class SentinelsMonitoringActor(
   seedSentinels: Seq[NodeAddress],
   config: MasterSlaveConfig,
   onInitFailure: Throwable => Unit,
-  onMasterChange: RedisNodeClient => Unit
+  onMasterChange: RedisNodeClient => Unit,
+  stateObserver: OptArg[SentinelStateObserver] = OptArg.Empty,
```

Comment:
That's a very specific `stateObserver`, I presume

Reply:
Changed to more specific `sentinelStateObserver`.